pom.xml

```
@@ -30,8 +30,8 @@
   </developers>
 
   <scm>
-    <connection>scm:git:ssh://github.com:jenkinsci/generic-webhook-trigger-plugin.git</connection>
-    <developerConnection>scm:git:git@github.com:jenkinsci/generic-webhook-trigger-plugin.git</developerConnection>
+    <connection>scm:git:https://github.com/jenkinsci/generic-webhook-trigger-plugin</connection>
+    <developerConnection>scm:git:https://github.com/jenkinsci/generic-webhook-trigger-plugin</developerConnection>
```

Causes https://github.com/jenkinsci/generic-webhook-trigger-plugin/pull/343#issuecomment-2637507717

```suggestion
    <developerConnection>scm:git:git@github.com:jenkinsci/generic-webhook-trigger-plugin.git</developerConnection>
```